### DISCUSSION TITLE
2008/09/28/ni-days

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#246] [Announcements]
# 2008/09/28/ni-days

公司又开始准备一年一度“NI Days”活动了。这是公司每年最重要的活动之一，是每年最大规模的与客户交流的活动。

https://blog.qizhen.xyz/2008/09/28/ni-days

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Zhang
2008年10月24日 14:02 编辑
一直到没有时间去看看，据说大部分是NI的推广活动。

回复
Nan
2008年10月3日 11:23 编辑
今年的NI Week和往年不同，增加了Robotics Summit. 我个人非常喜欢这个系列，从这个变化可以看出NI正在增加关于Robotics方面的侧重。
 
我不大清楚NI Week中国的内容是不是和Austin相同。

回复
杰
2008年09月29日 00:08 编辑
我参加了2004.2005.2006三年的NI DAYS，一次在北京，两次在上海。
后面没参加了，感觉雷同～呵呵，不好意思。
06年本来就已经不想去了，后来看宣传上说LabVIEW他爹要来演讲，于是下决心去了。印象中在香格里拉吧，感觉很好。
现在我做程序的时候，有时候搞不定，自言自语的指着屏幕说，小子别跟我拗着，再拗着叫你爹揍你，哈哈，还挺管用，一会就搞定了～

回复
